packages/compass-connections/src/components/connection-list/connection-list.tsx

```diff
@@ -127,26 +163,38 @@ function ConnectionList({
 }): React.ReactElement {
   const [recentHeaderHover, setRecentHover] = useState(false);
 
+  const { theme } = useTheme();
+
+  const useNewSidebar = process?.env?.COMPASS_SHOW_NEW_SIDEBAR === 'true';
+
+  const isExpanded = true; // TODO
```

Comment:
Is this a TODO for before this PR is merged?

Reply:
https://jira.mongodb.org/browse/COMPASS-5967

Will ammend.